qa/rpc-tests/abandonconflict.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env python2
+# Copyright (c) 2014-2015 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import *
+try:
+    import urllib.parse as urlparse
+except ImportError:
+    import urlparse
+
+class AbandonConflictTest(BitcoinTestFramework):
+
+    def setup_network(self):
+        self.nodes = []
+        self.nodes.append(start_node(0, self.options.tmpdir, ["-debug","-logtimemicros","-minrelaytxfee=0.00001"], binary="bitcoind"))
```

Comment:
I think we should drop the `binary="bitcoind"` part -- by default `start_node` can pick up the binary from the BITCOIND environment variable which is preferred when set.